### ISSUE TITLE
Sniffdog.. sniffs out!

### ISSUE THREAD

**Issue by rabi01** [Sniffdog/Sniffdogminer#191]
I have started using shiffdog miner. When I start mining, I found "Sniffdog.. sniffs out! as 12.43903 ±49%" and "Ccmineralexis-         6.10438 ±0%". 

What is the difference between both of them?

**Comment by Sniffdog** [comment]
Yeah i may have to remove that since not to many know about price variance and how price will swing wildly on a daily basis

**Comment by rabi01** [comment]
Its not about that, the amount it shows in the benchmark, the earning is significantly less than that. What do you think about this? The earning is no where near 6.10438
